templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -190,5 +190,126 @@
 				</div>
 			{{end}}
 		{{end}}
+
+		{{if .IssueDependenciesEnabled}}
+		<div class="ui divider"></div>
+
+		<div class="ui depending">
+			<span class="text"><strong>{{.i18n.Tr "repo.issues.dependency.title"}}</strong></span>
+			<br>
+			{{if .BlockedByDependencies}}
+			<span class="text">
+				{{if .Issue.IsPull}}
+					{{.i18n.Tr "repo.issues.dependency.issue_closing_blockedby"}}:
+				{{else}}
+					{{.i18n.Tr "repo.issues.dependency.pr_closing_blockedby"}}:
+				{{end}}
+			</span>
+			<div class="ui relaxed divided list">
+				{{range .BlockedByDependencies}}
+				<div class="item">
+					<div class="right floated content">
+						<a class="delete-dependency-button" onclick="deleteDependencyModal({{.Issue.ID}}, 'blockedBy');">
```

Comment:
Can you use `IssueDependency.IssueID`/`IssueDependency.DependencyID` for this? Also I think it would be cleaner to make two js functions and don't use hardcoded strings as parameter.

Reply:
But then I had two seperate functions which would be exactly the same (or call another function) exept for one string (which then would be set inside the function instead of setting it when i call the function)